doc/api/index.md

```diff
@@ -1853,25 +1857,149 @@ In _DirectFile_ mode (see [loadVideo
 options](./loadVideo_options.md#prop-transport)), returns ``undefined``.
 
 
+<a name="meth-setMinVideoBitrate"></a>
+### setMinVideoBitrate #########################################################
+
+_arguments_: ``Number``
+
+Set a minimum video bitrate reachable through adaptive streaming.
+
+When the bitrate is choosen through adaptive streaming (i.e., not enforced
+manually through APIs such as `setVideoBitrate`), the player will never switch
+to a video quality with a bitrate lower than that value.
+
+The exception being when no quality has a lower bitrate, in which case the
+maximum quality will always be chosen instead.
+
+For example, if you want that video qualities chosen automatically never have
```

Comment:
quality*

Reply:
I think that it works in that context.
Like: "if you want that [every] video qualities chosen automatically"

I might be wrong though

Reply:
Ok, I hadn't seen it like this.